tests/openai.test.ts

```diff
@@ -0,0 +1,118 @@
+import { expect, test } from '@playwright/test';
+
+import {
+	MOCK_OPENAI_COMPLETION_RESPONSE_1,
+	MOCK_OPENAI_MODELS,
+	MOCK_SESSION_1_RESPONSE_1,
+	mockOpenAICompletionResponse,
+	mockOpenAIModelsResponse
+} from './utils';
+
+test.describe('OpenAI Integration', () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/settings');
+	});
+
+	test('fetches data from OpenAI with a correct API Key', async ({ page }) => {
+		await mockOpenAIModelsResponse(page, MOCK_OPENAI_MODELS);
+		await expect(page.getByText('Sync was successful')).toBeVisible();
+	});
```

Comment:
Let's move `await expect(page.getByText('Sync was successful')).toBeVisible();` inside the `mockOpenAIModelsResponse()` function and remove this test.